executor/update.go

```diff
@@ -190,7 +193,9 @@ func (e *UpdateExec) exec(ctx context.Context, schema *expression.Schema, row, n
 		// Update row
 		changed, err1 := updateRecord(ctx, e.ctx, handle, oldData, newTableData, flags, tbl, false, e.memTracker)
 		if err1 == nil {
-			e.updatedRowKeys[content.Start].Set(handle, changed)
+			memDelta := e.updatedRowKeys[content.Start].Set(handle, changed)
+			memDelta += int64(handle.ExtraMemSize())
```

Comment:
If a handle is inserted multiple times, ExtraMemSize will be tracked many times... But in fact, the value in the map will be replace